jspsych/jspsych-contrib
# Addition of the Triple Handle Controller plugin for continuous, three-dimensional affective ratings

Hi all, this PR introduces the triple-handle controller plugin, enabling continuous three-dimensional affective ratings (e.g., valence, arousal, dominance) in jsPsych experiments. It lets participants adjust three separate handles in real time, providing dynamic affective data during stimulus presentation. 

# Included:

- Plugin code
- README
- package.json
- Extras: /docs with documentation, /examples with a working demo, and a test suite.

# How to verify:

You will need a handle controller. We developed and tested the plugin using a [Logitech Flight Throttle Quadrant](https://www.logitechg.com/en-us/products/flight/flight-simulator-throttle-quadrant.html).

- Try the Demo: Open the provided example file and interact with the three handles to see real-time ratings.
- Check Data: Inspect the console or saved data to confirm that ratings and timestamps are recorded as expected.


Feedback is welcome, especially on code clarity and documentation. Thanks for reviewing!
